target/linux/ath79/dts/ar9341_engenius_ens202ext.dts

```diff
@@ -0,0 +1,150 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+#include "ar9341.dtsi"
+
+/ {
+	model = "Engenius ENS202EXT";
+	compatible = "engenius,ens202ext", "qca,ar9341";
+
+	aliases {
+		serial0 = &uart;
+		led-boot = &system;
+		led-failsafe = &system;
+		led-running = &system;
+		led-upgrade = &system;
+	};
+
+	keys {
+		compatible = "gpio-keys-polled";
+		poll-interval = <20>;
```

Comment:
Please use gpio-keys instead and drop poll-interval.

Reply:
Just curious, whats the difference?

I was on v19.07.2 tag when I made the DTS, I suppose its outdated

Reply:
gpio-keys is using a different mechanism to detect pressed buttons, and we prefer that one: https://github.com/openwrt/openwrt/commit/8961d2268e3c196ed0bcdb7eded622c434982925

Now devices have been requested to use interrupt-driven gpio-keys for quite some time, but the old ones where only converted after 19.07 has been branched.

That's one of the many reason why you have to test this with master to get it in.